src/components/sbb-navigation-section/sbb-navigation-section.stories.js

```diff
@@ -133,7 +133,6 @@ const DefaultTemplate = (args) => [
       {navigationList('Label')}
       {navigationList('Label')}
 
-      {navigationList('Label')}
```

Comment:
What happens if we add some columns after the button? Does it render as expected in the image below, for example? Or should they simply not be added in this way?
<img width="1624" alt="image" src="https://user-images.githubusercontent.com/101181211/215439246-f676ba14-92a8-4860-a42c-684575eaa248.png">